packages/scalable-vega/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "scalable-vega",
```

Comment:
The name is already taken so you need to change it.

Reply:
any suggestions ?

Reply:
@leibatt?